README.md

```diff
@@ -70,39 +64,64 @@ interface AssumedRoleCredentials {
 }
 ```
 
-#### Get Profile
+```js
+blinkMobileIdentity.assumeAWSRole()
+  .then(credentials => {
+    // Use AWS credentials
+  });
+```
+
+### Get Profile
 
 ```js
 getProfile () => Promise{Auth0Profile}
 ```
 
+```js
+blinkMobileIdentity.getProfile()
+  .then(profile => {
+    // Use Auth0 profile
+  });
+```
+
 See [Auth0 Profile Structure](https://auth0.com/docs/user-profile/user-profile-structure) for available properties. 
 
-### 2. Extend Existing CLI
+### Manage Tenants
 
-Extending an existing CLI with login and logout commands will allow for these commands to be used from the command line
+Get and set the current tenant. Also get and remove list of previously used tenants.
 
-The login and logout commands internally call the login and logout functions from `BlinkMobileIdentity`
+```js
+getTenants () => Promise{Tenants}
+```
 
-#### Useage
+```js
+setTenant (tenantName: String) => Promise{Tenants}
+```
 
+```js
+removeTenant (tenantName: String) => Promise{Tenants}
 ```
-login                       => start the login process, if no flags are passed, a browser based login will begin
-  --username <username>     => username to login with, if password is not specified, you will be prompted for it
-  --password <password>	    => password to login with, requires the username flag as well
-  --email <email>           => email address to send code to for passwordless authentication
-  --sms <phone>             => phone number to send code to for passwordless authentication
 
-logout                      => logout of the service being extended
+```js
+interface Tenants {
+  current : String,
+  previous : Array{String}
```

Comment:
If order is unimportant and you need unique values, then perhaps look at using `Set` instead of `Array`. This should be available in Node 4.x